appveyor.yml

```diff
@@ -0,0 +1,26 @@
+version: .{build}-{branch}
+
+environment:
+
+  matrix:
+    - PYTHON: "C:\\Python27"
+      PYTHON_VERSION: "2.7.x" 
+      PYTHON_ARCH: "32"
+
+    - PYTHON: "C:\\Python27-x64"
+      PYTHON_VERSION: "2.7.x" 
+      PYTHON_ARCH: "64"
+
+install:
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "python --version"
+  - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  - "pip install --disable-pip-version-check --user --upgrade pip"
+
+build_script:
+  - "%CMD_IN_ENV% python setup.py install"
+
+
+test_script:
+  - "%CMD_IN_ENV% python setup.py test"
```

Comment:
you can add a command before calling ``setup.py test`` to install requirements from ``pip``. 

Rather than modifying ``setup.py``, replace ``schema_salad`` in ``requirements.txt`` to 
``https://github.com/kapilkd13/schema_salad/archive/windows#egg=schema_salad-2.4.20170612171942``